src/tasks/daily.ts

```diff
@@ -349,6 +398,12 @@ export function configureSnojo(): void {
 }
 
 export const DailyTasks: Task[] = [
+  {
+    name: "Chibi Buddy",
+    ready: () => have($item`ChibiBuddy™ (on)`) || have($item`ChibiBuddy™ (off)`),
+    completed: () => have($effect`ChibiChanged™`),
+    do: () => chibiSetup(),
```

Comment:
```suggestion
    do: chibiSetup,
```

Reply:
Should I change others to use this format instead as well? For example:

`  {
    name: "Check Barf Mountain Quest",
    ready: () => get("stenchAirportAlways") || get("_stenchAirportToday"),
    completed: () => !attemptCompletingBarfQuest,
    do: () => checkBarfQuest(),
  },`

Would instead just be `do: checkBarfQuest,`?

Reply:
ye

Reply:
Alright, I just did these updates in this PR as well, hopefully that is fine and not too distasteful